CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used, as specified in the issue template. Feature requests also have an issue template, containing
+questions that should be answered in the issue. We aim for 100% coverage of the Discord API; we will
+wait until a new feature is supported in the [Discord API documentation] before adding its support
+to Twilight. Before making an issue, be sure to consider joining the [Twilight Discord] and bringing
+up your topic in the `#support` channel.
+
+# Pull Requests
+
+Pull requests must be named with the format `{crate}: {short description of change}`, and should use
+lower case letters. If the change spans more than one crate, separate the crate names with a comma
+and a space: `{crate1}, {crate2}: {short description of change}`. Pull requests must be made from a
+new branch. Please avoid making pull requests from the `trunk` branch. If adding a feature or
+enhancement, use the term `add` or something sufficiently similar. If fixing a bug, use the term
+`fix`, or something sufficiently similar. Avoid force-pushing to a pull request branch, as this
+erases review comment history.
+
+Contributors should add tests and documentation that reflects their changes.
+
+## Tests
+
+Contributors should write tests to ensure soundness in present and future code. Alongside regular
```

Comment:
As noted in Discord, this should be `correctness` and not `soundness` since we aren't writing any unsafe :eyes: